package ai.dnai.io.pseudorandomfilegenerator.api.exceptions

class WrongNumberOfSuppliersException(message: String) : GeneralCustomException(message) {
}